ereg-mapper-core/src/main/java/no/nav/registre/ereg/mapper/EregMapper.java

```diff
@@ -61,7 +61,7 @@ public String mapEregFromRequests(
                                         if (d.getNaeringskode() == null) {
                                             NaeringskodeRecord randomNaeringskode = nameService.getRandomNaeringskode();
                                             String dato = getDateNowFormatted();
-                                            if (!"".equals(randomNaeringskode.getValidFrom())) {
+                                            if (!"" .equals(randomNaeringskode.getValidFrom())) {
```

Comment:
Er ikke dette litt rar formatering? Med mellomrom etter ""? Blir ihvertfall litt støy i pull-requester hvis dette endres fram og tilbake for hver PR folk legger ut.

Reply:
Ja dette er rart :P Skal fikses